assets/templates/partials/calendar/filter/census.tmpl

```diff
@@ -1,42 +1,21 @@
-<form>
+<form class="filters__release-census">
+  {{ template "partials/calendar/hidden-inputs/keywords" . }}
+  {{ template "partials/calendar/hidden-inputs/release-types" . }}
+  {{ template "partials/calendar/hidden-inputs/sort-mode" . }}
+  {{ template "partials/calendar/hidden-inputs/before-date" . }}
+  {{ template "partials/calendar/hidden-inputs/after-date" . }}
```

Comment:
The above is correct, but this same pattern of including hidden inputs for all other forms except for this form's inputs also exists on all the other forms in order to carry the complete state of the system back to the server when a form is submitted.

Whatever the state of the census flag, this has to be carried through by the keyword search when that is submitted, the sort when that is submitted, etc. A hidden input for the census flag should be created, and added to all the other forms on the page in order achieve this.